src/Header/NavMobileItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC, ReactNode } from 'react';
+import { styled } from '@storybook/theming';
+import { color, spacing } from '../_tokens';
+import { Icon, IconType } from '../Icon/Icon';
+import { Text } from '../Text';
+
+export interface DesktopItemProps {
+  icon?: IconType;
+  iconColor?: keyof typeof color;
+  customIcon?: ReactNode;
+  title: string;
+}
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: ${spacing[3]};
+  height: ${spacing[8]};
+  align-items: center;
+`;
+
+const IconWrapper = styled.div`
+  width: ${spacing[4]};
+  padding-top: ${spacing[0.25]};
+`;
+
+const TextWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${spacing[0.5]};
+`;
+
+export const NavMobileItem: FC<DesktopItemProps> = ({
+  icon,
+  iconColor,
+  customIcon,
+  title,
+}) => {
+  return (
+    <Container>
+      <IconWrapper>
+        {!customIcon && icon && (
+          <Icon name={icon} size={16} color={iconColor} />
```

Comment:
All these `icons` and `customIcons` should have `alt=""` 

https://www.w3.org/WAI/tutorials/images/decorative/#example-2-decorative-image-as-part-of-a-text-link

Reply:
These are SVGs. Alt are applied only on images, no?

Reply:
You'll need to set `aria-hidden="true"` on SVGs. See how it shows up as empty `img` in the a11y tree:

<img width="373" alt="CleanShot 2023-05-09 at 15 41 29@2x" src="https://github.com/chromaui/tetra/assets/42671/08c7edfd-671f-4a77-b9db-561008612df2">

Compare that to this example where I've set aria-hidden 
<img width="385" alt="CleanShot 2023-05-09 at 15 42 51@2x" src="https://github.com/chromaui/tetra/assets/42671/4731dd56-80bf-48b4-985f-33be81b75508">

Reply:
We have that recommendation in the SDS docs: https://master--5ccbc373887ca40020446347.chromatic.com/?path=/docs/icon--docs

Reply:
Ok that's done. All icons are now using `aria-hidden`